src/scss/_variables.scss

```diff
@@ -13,4 +13,12 @@ $o-card-spacing-unit: 10px;
 
 /// Size of image when left aligned
 /// @type {Number}
-$o-card-image-width: 100px;
+$o-card-image-width: 40%;
+
+/// Colour of links and tags on Opinion cards
+/// @type {String}
+$o-card-opinion-colour: #416e92; // This should be put into o-colors
+
+$o-card-headshot-colour: #d0e2e7; // This should be put into o-colors
```

Comment:
Probably before this gets merged. As you mentioned I need to make sure I know where these are coming from and whether they should be put into `o-colors` or whether they should be standardised.